packages/contracts-bedrock/src/governance/GovernanceTokenInterop.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import { ERC20Votes } from "@openzeppelin/contracts/token/ERC20/extensions/ERC20Votes.sol";
+import { Predeploys } from "src/libraries/Predeploys.sol";
+import { IGovernanceDelegation } from "src/governance/IGovernanceDelegation.sol";
+import { GovernanceToken } from "src/governance/GovernanceToken.sol";
+
+contract GovernanceTokenInterop is GovernanceToken {
+    /// @notice Returns the checkpoint for a given account at a given position.
+    /// @param _account The account to get the checkpoints for.
+    /// @param _pos     The psition to get the checkpoints at.
+    /// @return         The checkpoint at the given position.
+    function checkpoints(address _account, uint32 _pos) public view override(ERC20Votes) returns (Checkpoint memory) {
+        if (_migrated(_account)) {
+            return IGovernanceDelegation(Predeploys.GOVERNANCE_DELEGATION).checkpoints(_account, _pos);
+        } else {
+            return super.checkpoints(_account, _pos);
+        }
+    }
+
+    /// @notice Returns the number of checkpoints for a given account.
+    /// @param _account The account to get the number of checkpoints for.
+    /// @return         The number of checkpoints for the given account.
+    function numCheckpoints(address _account) public view override(ERC20Votes) returns (uint32) {
+        if (_migrated(_account)) {
```

Comment:
Based on the description in https://github.com/ethereum-optimism/specs/pull/342, this PR is out of date?

Reply:
yes, now that PR is merged, im updating this one

Reply:
updated now, CI is failing from a tests in liquidity migration contracts, see [here](https://discord.com/channels/1244729134312198194/1253359437838684170/1276802566474301451)